db/migrate/20170720194715_create_middleware_jdr_reports.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMiddlewareJdrReports < ActiveRecord::Migration[5.0]
+  def change
+    create_table :middleware_jdr_reports do |t|
```

Comment:
I don't care what you call the table if it does not drive what's presented in the UI. In the UI we want to call it JDR or JBoss Diagnostic Report.